kevm-pyk/src/kevm_pyk/kompile.py

```diff
@@ -106,6 +107,28 @@ def kevm_kompile(
                 )
                 return kompile(output_dir=output_dir, debug=debug, verbose=verbose)
 
+            case KompileTarget.MAUDE:
+                kompile_maude = MaudeKompile(
+                    base_args=base_args,
+                )
+                kompile_haskell = HaskellKompile(base_args=base_args)
```

Comment:
I'm unfamiliar with how the Maude backend works, but from this, I understand that it depends on the Haskell backend similar to what the booster backend does.

Reply:
For the time being, the Maude backend does not implement `get-model` or `check-implication`, so it still requires the Haskell backend to implement those endpoints. This will eventually change, but that's not needed for a minimum viable product, so it has not been the focus.